change/@fluentui-react-tooltip-40830067-8e20-4a61-9deb-91660edd6e2c.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

```suggestion
  "type": "patch",
```

Patch? Does not seem like a real feature IMO